zenodo/modules/profiles/templates/zenodo_profiles/orcid_profile.html

```diff
@@ -0,0 +1,221 @@
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+-#}
+
+{% extends "zenodo_theme/page.html" %}
+
+{%- block page_body %}
+<div class="container">
+  <div class="row" id="orcid-profile">
+    <div class="col-sm-3 col-xs-12" id="profile" style="margin-bottom:20px">
+      <br />
+      <img height="400px" width="400px"
+        alt="Profile Picture" class="img-responsive img-rounded center-block"
+        src="https://members.orcid.org/sites/default/files/vector_iD_icon.svg"
+        alt="Profile Picture" />
+      <div id="orcid_profile">
+        <div id="researcher_name">
+          <h1 style="line-height:0.8">
+            <strong></strong>
+            <br />
+            <small id="researcher_affiliation"></small>
+          </h1>
+        </div>
+        <div id="researcher_orcid" style="margin-bottom: 4px">
+          <img width="26px" style="vertical-align:middle; margin-left: 5px" src="{{ url_for('static', filename='img/orcid.png') }}" alt="Orcid" />
+          <a class="no_hover" href="https://orcid.org/{{orcid_id}}"><span style="margin-left: 3px;font-size: 16px">{{ orcid_id }}</span></a>
+        </div>
+        <div id="researcher_bio"><h3><small></small></h3></div>
+        <div id="researcher_address">
+          <hr />
+          <span class="fa fa-map-marker fa-fw" aria-hidden="true"></span>
+          <span class="content"></span>
+        </div>
+      </div>
+      <div class="loader">
+      </div>
+      <br />
+    </div> <!-- .col-sm-3 .col-xs-12 -->
+
+    <div class="col-sm-9 col-xs-12">
+      <h1>Popular Uploads</h1>
+      {% include "zenodo_profiles/results.html" %}
+      <p class="text-center">
+        <a class="btn btn-default" href="{{ url_for('zenodo_profiles.profile_search', orcid_id=orcid_id) }}">More</a>
+      </p>
+    </div> <!-- .col-sm-9 .col-xs-12 -->
+  </div> <!-- .row -->
+</div> <!-- .container -->
+{%- endblock page_body %}
+
+{%- block javascript %}
+{% assets "zenodo_theme_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
+{% assets "zenodo_search_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
+<script>
+$('#orcid_profile, #researcher_name, #researcher_orcid, #researcher_bio, #researcher_address, #researcher_affiliation').hide();
+
+$.ajax({
+  url: 'https://pub.orcid.org/v2.1/{{orcid_id}}/activities',
+  headers: {'Accept': 'application/json'},
+  contentType: 'application/json',
+  dataType: 'json',
+  success: function(result){
```

Comment:
It's better to extract this logic into some separate function, since now it looks pretty complex. There are also some utility functions available I believe that might simplify many parts of the code (eg. [Lodash's `_.map`](https://stackoverflow.com/questions/35136306/what-happened-to-lodash-pluck) for extracting deeply nested values from response objects).